docs/PLUGIN_CATALOG.md

```diff
@@ -79,6 +79,7 @@ This file is automatically generated. If you would like to add to the plugin lis
 | [USE](https://github.com/intelsdi-x/snap-plugin-collector-use) | [Intel](https://github.com/intelsdi-x) | Collect Linux Utilization, Saturation and Errors metrics | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-use.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-use) | [release](https://github.com/intelsdi-x/snap-plugin-collector-use/releases) |
 | [Users](https://github.com/intelsdi-x/snap-plugin-collector-users) | [Intel](https://github.com/intelsdi-x) | Collects Linux user metrics from utmp | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-users.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-users) | [release](https://github.com/intelsdi-x/snap-plugin-collector-users/releases) |
 | [Yarn](https://github.com/intelsdi-x/snap-plugin-collector-yarn) | [Intel](https://github.com/intelsdi-x) | Collects metrics from Hadoop Yarn | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-yarn.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-yarn) | [release](https://github.com/intelsdi-x/snap-plugin-collector-yarn/releases) |
+| [Vsphere](https://github.com/intelsdi-x/snap-plugin-collector-vsphere)<br />[![supported_label]](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_STATUS.md#supported-plugins) | [Intel](https://github.com/intelsdi-x) | Collects performance counters data for VMWare vSphere cluster | | |
```

Comment:
@ZhengQingyuan, could you add build status and link to release? It would look like below (ready to use):
```
| [Vsphere](https://github.com/intelsdi-x/snap-plugin-collector-vsphere)<br />[![supported_label]](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_STATUS.md#supported-plugins) | [Intel](https://github.com/intelsdi-x) | Collects performance counters data for VMWare vSphere cluster | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-vsphere.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-vsphere) | [release](https://github.com/intelsdi-x/snap-plugin-collector-vsphere/releases) |
```